core/src/main/java/org/geysermc/geyser/registry/populator/BlockRegistryPopulator.java

```diff
@@ -301,6 +303,12 @@ private static void registerBedrockBlocks() {
 
             builder.commandBlock(commandBlockDefinition);
 
+            if (mobSpawnerBlockDefinition == null) {
+                throw new AssertionError("Unable to find mob spawner block in palette");
+            }
+
+            builder.mobSpawnerBlock(mobSpawnerBlockDefinition);
```

Comment:
I would just remove the extra space of mobSpawner and commandBlock to match water/air/movingBlock below